manager/ui/war/gulpfileCI.js

```diff
@@ -0,0 +1,80 @@
+// Gulp File specifically for Cypress CI
+
+const { gulp, connect } = require('./gulpfile.js');
```

Comment:
this gulpfile is for CI only

it turned out to be simpler just to have a separate one that overrides the first.